<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraOne</class>
 <widget class="QDialog" name="CameraOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="cam1lbl">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cam4lbl">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cam3lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cam2lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cam5lbl">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="leftView_Box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Drive Aim</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rightView_Box">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Drive Aim</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
